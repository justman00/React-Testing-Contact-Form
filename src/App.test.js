import React from "react";
import { render, screen } from "@testing-library/react";
import App from "./App";

test("renders App without errors", () => {
  render(<App />);
});

test("renders styles accordingly", () => {
  const { wrapper, container } = render(<App />);
  console.log(wrapper);
  expect(container).toMatchSnapshot();
});
