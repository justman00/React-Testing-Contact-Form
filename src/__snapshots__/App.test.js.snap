// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders styles accordingly 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="App"
    >
      <form>
        <div>
          <label
            for="firstName"
          >
            First Name*
          </label>
          <input
            id="firstName"
            name="firstName"
            placeholder="Edd"
          />
        </div>
        <div>
          <label
            for="lastName"
          >
            Last Name*
          </label>
          <input
            id="lastName"
            name="lastName"
            placeholder="Burke"
          />
        </div>
        <div>
          <label
            for="email"
          >
            Email*
          </label>
          <input
            id="lastName"
            name="email"
            placeholder="bluebill1049@hotmail.com"
          />
        </div>
        <div>
          <label
            for="message"
          >
            Message
          </label>
          <textarea
            id="message"
            name="message"
          />
        </div>
        <input
          type="submit"
        />
      </form>
    </div>
  </div>
</div>
`;
